opium/src/opium.mli

```diff
@@ -1,3 +1,9 @@
+module Private : sig
+  module Router : module type of struct
```

Comment:
Just curious, `module type of Router` wouldn't be the same thing? Or is this to allow adding other elements to the signature?

Reply:
I recall that the simplest form omitted some type equalities for types inside `Router.t`. So if one used `Router` from outside this alias, the type `Router.t` wouldn't be equal. But I think this bug was fixed not too long ago. I'll see if the simpler form works as well.